dds_registration/models.py

```diff
@@ -122,7 +138,7 @@ class Event(models.Model):
     @property
     def can_register(self):
         today = date.today()
-        return self.registration_open >= today and (not self.registration_close or today <= self.registration_close)
```

Comment:
It was my careless mistake.

Reply:
See above, we can assume that `registration_close` exists